File: c++/Project_algorithm/Baekjoon10886.cpp
#include <stdio.h>

int main() {
	int input = 0;
	scanf("%d", &input);
	int ycnt = 0, ncnt = 0;
	for (int i = 0; i < input; i++) {
		int n = 0;
		scanf("%d", &n);
		if (n == 0) ycnt++;
		else ncnt++;
	}
	printf(ncnt > ycnt ? "Junhee is cute!" : "Junhee is not cute!");
	return 0;
}

File: Java/src/Baekjoon/Baekjoon10886.java
package baekjoon;

import java.util.Scanner;

public class Baekjoon10886 {
    public static void main(String[] args) throws Exception {
        Scanner scanner = new Scanner(System.in);
        int input = scanner.nextInt();
        int ycnt = 0, ncnt = 0;
        for (int i = 0; i < input; i++) {
            int input_cute = scanner.nextInt();
            if (input_cute == 0) ycnt++;
            else ncnt++;
        }
        scanner.close();
        System.out.println(ncnt > ycnt ? "Junhee is cute!" : "Junhee is not cute!");
    }
}
